packages/serverless/src/apollo/app/collection-index.ts

```diff
@@ -100,28 +95,22 @@ const indexExist = authorizeWrapper(
 const indexCreate = authorizeWrapper(
   IndexCreateRequest,
   async (_root: unknown, args: TIndexCreateRequest, ctx) =>
-    withTransaction((session) =>
-      createIndex(
-        args.databaseName,
-        ctx.userName,
-        args.collectionName,
-        args.indexField,
-        session
-      )
+    createIndex(
```

Comment:
why did you drop withTransaction HOF? Does it need to use session?

Reply:
We cannot create index withing transaction